### PR TITLE
Odd display on error type when multiple mimes for same extension

### PR DESCRIPTION
**Pull request by cavasinf**
According to https://github.com/pqina/filepond-plugin-file-validate-type/issues/44

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -129,7 +129,7 @@ const plugin = ({ addFilter, utils }) => {
                             sub: replaceInString(
                                 query('GET_FILE_VALIDATE_TYPE_LABEL_EXPECTED_TYPES'),
                                 {
-                                    allTypes: acceptedFileTypesMapped.join(', '),
+                                    allTypes: [...new Set(acceptedFileTypesMapped.join(', '))],
```
Comment (cavasinf) [inline src/js/index.js:132]:
```suggestion
                                    allTypes: [...new Set(acceptedFileTypesMapped)].join(', '),
```

**Comment by cavasinf** [comment]
@rikschennink is it correct ?

**Comment by rikschennink** [comment]
@cavasinf I haven't gotten around to it

**Comment by rikschennink** [comment]
FilePond core still supports IE11 but IE11 doesn't support `new Set(iterable)` so I think we have to use fallback to create a unique set.

File: src/js/index.js [hunk 2]
```diff
@@ -123,15 +123,17 @@ const plugin = ({ addFilter, utils }) => {
                         )
                     ).filter(label => label !== false);
 
+                    const acceptedFileTypesMapped_unique = [...new Set(acceptedFileTypesMapped)];
```
Comment (cavasinf) [inline src/js/index.js:126]:
```suggestion
                    const acceptedFileTypesMapped_unique = acceptedFileTypesMapped.filter(function(item, index) {return acceptedFileTypesMapped.indexOf(item) === index});  
```

**Comment by cavasinf** [comment]
@rikschennink Oh god IE11.. Lot of respect to have this module runnable in IE11.

I've change the code to allow IE11 👍

**Pull request merged**